airflow/models/__init__.py

```diff
@@ -193,70 +194,6 @@ def get_fernet():
 _CONTEXT_MANAGER_DAG = None
 
 
-def clear_task_instances(tis,
```

Comment:
Is moving this related to this PR? Not opposed to it per se, just wondering how its related to these changes. If it isn't, I'd suggest leaving it for another (smaller) PR, since this one is already pretty big.

Reply:
I must say, I'm not sure anymore. Mostly I try to move unrelated changes to a new branch. So I think think there was a reason for this. Mind to just keep it as it is?